app/screens/draft_scheduled_post_options/send_draft.test.tsx

```diff
@@ -168,4 +179,37 @@ describe('Send Draft', () => {
         expect(deleteScheduledPost).toHaveBeenCalledWith('https://server.com', 'post_id');
         expect(removeDraft).not.toHaveBeenCalled();
     });
+
+    it('should proceed with sending when websocket is connected', () => {
```

Comment:
We no longer consider the websocket connection, so this tests is either not needed or has a wrong name.

Reply:
No loanger needed, will remove it.